rebar.config

```diff
@@ -0,0 +1,31 @@
+{erl_opts,
+ [
+  {parse_transform, lager_transform},
+  {src_dirs, ["src"]},
+  warn_unused_vars,
+  warn_export_all,
+  warn_shadow_vars,
+  warn_unused_import,
+  warn_unused_function,
+  warn_bif_clash,
+  warn_unused_record,
+  warn_deprecated_function,
+  warn_obsolete_guard,
+  strict_validation,
+  warn_export_vars,
+  warn_exported_vars,
+  warn_missing_spec,
+  warn_untyped_record, debug_info
+ ]
+}.
+{deps_dir, "deps"}.
+{deps,
+ [
+  {lager,  "2.*", {git, "git://github.com/basho/lager.git",     "2.1.0"}},
+  {eper,   "0.*", {git, "git://github.com/massemanet/eper.git", "0.90.0"}},
+  {sync,   "0.*", {git, "git://github.com/inaka/sync.git",      "0.1"}},
+  {getopt, "0.*", {git, "https://github.com/jcomellas/getopt",  "v0.8.2"}}
+ ]
+}.
+{escript_name, "xref_runner"}.
```

Comment:
What if we call it `xrefr` and we create an xrefr module with all the new code, instead of modifying `xref_runner`?